cluster/images/Dockerfile

```diff
@@ -1,3 +1,17 @@
+# Copyright 2023 The Karmada Authors.
```

Comment:
The year in the copyright should be the time when this file was created, right?

Reply:
fixed

Reply:
Amazing!
Just out of curiosity. How did you do that? Do it manually or by some automation?

Reply:
By automation! I wrote a script to traverse all target files and use 
```shell
 # use git log to get the year of the file creation
 year=$(git log --diff-filter=A --follow --format=%aD -- "$file" | head -1 | cut -d' ' -f4)
 # update the file with license
 $ADDLICENSE_BIN -f "$REPO_PATH/hack/boilerplate/LICENSE_TEMPLATE" -v -y "$year" "$file"
```
to get the year the file was created, and use addlicense to add license~ You can see more information in update-license.sh